types/stripe-js/elements-group.d.ts

```diff
@@ -967,48 +967,114 @@ export interface Appearance {
 
     // Spacing
     spacingUnit?: string;
+    gridRowSpacing?: string;
+    gridColumnSpacing?: string;
+    tabSpacing?: string;
+    accordionItemSpacing?: string;
+    /** @deprecated Use gridRowSpacing instead. */
     spacingGridRow?: string;
+    /** @deprecated Use gridColumnSpacing instead. */
     spacingGridColumn?: string;
+    /** @deprecated Use tabSpacing instead. */
     spacingTab?: string;
+    /** @deprecated Use accordionItemSpacing instead. */
     spacingAccordionItem?: string;
 
     // Colors
     colorPrimary?: string;
-    colorPrimaryText?: string;
     colorBackground?: string;
-    colorBackgroundText?: string;
     colorText?: string;
     colorSuccess?: string;
-    colorSuccessText?: string;
     colorDanger?: string;
-    colorDangerText?: string;
     colorWarning?: string;
-    colorWarningText?: string;
 
     // Text variations
     colorTextSecondary?: string;
     colorTextPlaceholder?: string;
 
+    // Accessible text
+    accessibleColorOnColorPrimary?: string;
+    accessibleColorOnColorBackground?: string;
+    accessibleColorOnColorSuccess?: string;
+    accessibleColorOnColorDanger?: string;
+    accessibleColorOnColorWarning?: string;
+    /** @deprecated Use accessibleColorOnColorPrimary instead. */
+    colorPrimaryText?: string;
+    /** @deprecated Use accessibleColorOnColorBackground instead. */
+    colorBackgroundText?: string;
+    /** @deprecated Use accessibleColorOnColorSuccess instead. */
+    colorSuccessText?: string;
+    /** @deprecated Use accessibleColorOnColorDanger instead. */
+    colorDangerText?: string;
+    /** @deprecated Use accessibleColorOnColorWarning instead. */
+    colorWarningText?: string;
+
     // Icons
+    iconColor?: string;
+    iconHoverColor?: string;
+    iconCardErrorColor?: string;
+    iconCardCvcColor?: string;
+    iconCardCvcErrorColor?: string;
+    iconCheckmarkColor?: string;
+    iconChevronDownColor?: string;
+    iconChevronDownHoverColor?: string;
+    iconCloseColor?: string;
+    iconCloseHoverColor?: string;
+    iconLoadingIndicatorColor?: string;
+    iconMenuColor?: string;
+    iconMenuHoverColor?: string;
+    iconPasscodeDeviceColor?: string;
+    iconPasscodeDeviceHoverColor?: string;
+    iconPasscodeDeviceNotificationColor?: string;
```

Comment:
These are net-new declarations, as we appear to have never added these to our types.